### DISCUSSION TITLE
OnlyFans 5.3 issue.

### DISCUSSION THREAD

**Discussion by funjunkstorage** [UltimaHoarder/UltimaScraper#1507] [General]
I have put in all of the cookie credentials and I get back this message. I am using version 5.3, it used to work for me. Is 5.3 still supported or does it no longer work? Any idea why I am getting this message?

Site: 0 = onlyfans | 1 = starsavn | 2 = fourchan | 3 = bbwchan | x = Exit
0
Auth (V1)
Auth Attempt 1/10
Auth Attempt 2/10
Auth Attempt 3/10
Auth Attempt 4/10
Auth Attempt 5/10
Auth Attempt 6/10
Auth Attempt 7/10
Auth Attempt 8/10
Auth Attempt 9/10
Auth Attempt 10/10
Auth (V2)
Auth Attempt 1/10
Auth Attempt 2/10
Auth Attempt 3/10
Auth Attempt 4/10
Auth Attempt 5/10
Auth Attempt 6/10
Auth Attempt 7/10
Auth Attempt 8/10
Auth Attempt 9/10
Auth Attempt 10/10
There's nothing to scrape.

**Comment by LunarPenguin** [comment]
Version 5.3 is in 6 days exactly a year old and is no longer supported.

**Comment by undacovabrutha** [comment]
I'm having the same issue.

**Reply by undacovabrutha** [reply]
How do I update?

**Reply by LunarPenguin** [reply]
if you know how to use git use that to update, otherwise download the zip file for the latest version from [here](https://github.com/DIGITALCRIMINAL/OnlyFans/releases) and replace your existing version with the contents of that zip file.

Make a backup of `auth.json` and `config.json` to prevent losing your settings

**Comment by undacovabrutha** [comment]
Thanks so much for responding. I was able to get updated and moving forward. Only thing now is only the avatar and headed are the only two files downloading. I’ll keep at it. But again, thanks!

Will

**Reply by LunarPenguin** [reply]
>Thanks so much for responding. I was able to get updated and moving forward.

good to hear you got the script updated.

>Only thing now is only the avatar and headed are the only two files downloading. I’ll keep at it

Yes i would suggest waiting a little bit before scraping again to see if the potential problem has resolved itself

**Reply by undacovabrutha** [reply]
![image](https://user-images.githubusercontent.com/79873133/129607191-3fad6467-86f3-4489-be89-ac3201a34560.jpeg)

**Reply by undacovabrutha** [reply]
This is the error message I'm getting. What am I doing wrong?

**Reply by junjuiceee** [reply]
I'm having the same issue. I created a thread but no substantive responses so far

**Reply by LunarPenguin** [reply]
it seems that version `4.62.0` of `tqdm` is not entirely stable yet and people in #1561 are reporting that version `4.59.0` does work so for a possible solution i would recommended uninstalling version `4.62.0` and installing version `4.59.0` of `tqdm` and see if that solves the problem you're having
